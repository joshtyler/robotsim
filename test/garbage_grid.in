adsfasf sdfasf 1 3 4
