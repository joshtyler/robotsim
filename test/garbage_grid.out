ERROR: Could not parse initial coordinates
